packages/models/src/lib/notifications/notification.model.ts

```diff
@@ -0,0 +1,27 @@
+import { NotificationKind } from './notification-kind';
+
+export interface Notification {
+    _id: string;
+
+    /**
+     * The ID of the user this notification was sent to.
+     */
+    destinationUserId: string;
+
+    /**
+     * The ID of the thing (Work, Document, Blog, etc) that triggered this notification.
+     */
+    sourceId: string;
+    
+    /**
+     * The ID of the notification source's parent (i.e. if the notification
+     * was triggered by a Section, this will be the parent Work's ID).
+     * Undefined if the source doesn't have a parent.
+     */
+    sourceParentId?: string | undefined;
```

Comment:
The idea behind this `sourceParentId` is that the source of a "new section" notification would be the work. However, that work has a parent (a user) which we probably also want to be able to show in the notification UI.
Ditto new comments (actual parent is a section, or possibly a work, but the parent of that is either a work or a user), etc , etc.